docs/SETUP/BACK_END.md

```diff
@@ -2,55 +2,72 @@
 
 ## Overview
 
-If you are planning to contribute to the data wrangling and database management on this project and will need to run the Python script, follow the installation and setup instructions below.
+If you plan to contribute to the data wrangling and database management on this project and need to run the Python script, follow the installation and setup instructions below.
 
 ## Setup
 
 ### Fork the Repository
 
-1. Navigate to [our GitHub repository](https://github.com/CodeForPhilly/vacant-lots-proj).
-2. Create a fork of the repository by clicking the "Fork" button in the top right corner of the page. This will create a copy of the repository in your own GitHub account.
+1. Navigate to [our GitHub repository](https://github.com/CodeForPhilly/clean-and-green-philly).
+2. Create a fork of the repository by clicking the "Fork" button in the top right corner of the page.
 3. Clone your fork of the repository to your local machine using `git clone`.
 
-Note: make sure to keep your fork up to date with the original repository by following the instructions [here](https://docs.github.com/en/get-started/quickstart/fork-a-repo#keep-your-fork-synced).
+**Note:** Keep your fork up to date with the original repository by following the instructions [here](https://docs.github.com/en/get-started/quickstart/fork-a-repo#keep-your-fork-synced).
 
 ### Set Environment Variables
 
-The project needs certain specific and sensitive pieces of information to run.  We store these in the user's development environment and not in source control.
+The project needs certain specific and sensitive pieces of information to run. We store these in the user's development environment and not in source control.
 
-For Mac and Linux, you can permanently store the environmental variables in your command line shell's configuration file, e.g. `~/.bashrc`, `~/.bash_profile`, `~/.zshrc` or `~/.profile`.  Add a line `export VAR_NAME=VALUE` in your file and run `source file` to read it in when newly created.  Any new shells will automatically have the new environment.
```

Comment:
This section should be left in.  It is convenient to have shell variables for some users.